regcore/migrations/0010_auto_20160322_1704.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def forward(apps, schema_editor):
+    schema_editor.execute("""
```

Comment:
It does so long as the two migrations are run at the same time. Migrations can be applied (and reversed) incrementally, however, so it's a safer bet to be cautious.